gitlab/metadata.csv

```diff
@@ -1,98 +1,165 @@
+gitlab.transaction.event_remote_mirrors_running_total,count,,,,[OpenMetrics V1] The counter for running remote mirrors,0,gitlab,remote mirrors running,
+gitlab.transaction.event_remote_mirrors_running.count,count,,,,[OpenMetrics V2] The counter for running remote mirrors,0,gitlab,remote mirrors running,
+gitlab.transaction.event_remove_branch_total,count,,,,[OpenMetrics V1] The counter when a branch is removed for any repository,0,gitlab,branches removed,
+gitlab.transaction.event_remove_branch.count,count,,,,[OpenMetrics V2] The counter when a branch is removed for any repository,0,gitlab,branches removed,
+gitlab.transaction.event_remove_repository_total,count,,,,[OpenMetrics V1] The counter when a repository is removed,0,gitlab,repos removed,
+gitlab.transaction.event_remove_repository.count,count,,,,[OpenMetrics V2] The counter when a repository is removed,0,gitlab,repos removed,
+gitlab.transaction.event_remove_tag_total,count,,,,[OpenMetrics V1] The counter when a tag is remove for any repository,0,gitlab,tags removed,
+gitlab.transaction.event_remove_tag.count,count,,,,[OpenMetrics V2] The counter when a tag is remove for any repository,0,gitlab,tags removed,
+gitlab.transaction.event_sidekiq_exception_total,count,,,,[OpenMetrics V1] The counter of Sidekiq exceptions,-1,gitlab,sidekiq exceptions,
+gitlab.transaction.event_sidekiq_exception.count,count,,,,[OpenMetrics V2] The counter of Sidekiq exceptions,-1,gitlab,sidekiq exceptions,
+gitlab.transaction.event_stuck_import_jobs_total,count,,,,[OpenMetrics V1] The count of stuck import jobs,-1,gitlab,stuck import jobs,
+gitlab.transaction.event_stuck_import_jobs.count,count,,,,[OpenMetrics V2] The count of stuck import jobs,-1,gitlab,stuck import jobs,
+gitlab.transaction.event_update_build_total,count,,,,[OpenMetrics V1] The counter for update build for API /jobs/request/:id,0,gitlab,update build,
+gitlab.transaction.event_update_build.count,count,,,,[OpenMetrics V2] The counter for update build for API /jobs/request/:id,0,gitlab,update build,
+gitlab.transaction.new_redis_connections_total,count,,connection,,[OpenMetrics V1] The counter for new Redis connections,0,gitlab,new redis connections,
+gitlab.transaction.new_redis_connections.count,count,,connection,,[OpenMetrics V2] The counter for new Redis connections,0,gitlab,new redis connections,
+gitlab.transaction.queue_duration_total,count,,,,[OpenMetrics V1] The duration jobs were enqueued before processing,0,gitlab,jobs enqueued processing,
+gitlab.transaction.queue_duration.count,count,,,,[OpenMetrics V2] The duration jobs were enqueued before processing,0,gitlab,jobs enqueued processing,
+gitlab.transaction.rails_queue_duration_total,gauge,,,,[OpenMetrics V1] The latency between GitLab Workhorse forwarding a request to Rails,0,gitlab,rails queue duration,
+gitlab.transaction.rails_queue_duration.count,gauge,,,,[OpenMetrics V2] The latency between GitLab Workhorse forwarding a request to Rails,0,gitlab,rails queue duration,
```

Comment:
If this is a `gauge`, the `.count` suffix looks off. Or maybe the type is not right? There are other examples in this file with this issue. Bear in mind that the `_total` suffix is only dropped from the name for counters, not for gauges (unless I'm misremembering).

Reply:
Oh nice catch I missed that. This is supposed to be a counter

Reply:
It's not a counter, it's a gauge (confirmed with the payload returned by the endpoint). Fixed in https://github.com/DataDog/integrations-core/pull/14273/commits/723b62915a8fded0f43c8ba363e8e01a155d4ad8